spec/api-app-spec.js

```diff
@@ -157,6 +157,24 @@ describe('app module', () => {
         done()
       })
     })
+
+    it('exits gracefully on macos', (done) => {
```

Comment:
this arrow function should be changed to `function(done)` otherwise the implicit `this` will prevent the `this.skip()` below from working properly